content/en/blog/_posts/2022-12-16-statefulset-migration.md

```diff
@@ -0,0 +1,205 @@
+
+In this demo, you'll use the `StatefulSetStartOrdinal` feature to migrate a
+StatefulSet from one Kubernetes cluster to another. For this demo, the
+[redis-cluster](https://github.com/bitnami/charts/tree/main/bitnami/redis-cluster)
+Bitnami Helm chart is used to install Redis.
+
+Tools Required:
+ * [yq](https://github.com/mikefarah/yq)
+ * [helm](https://helm.sh/docs/helm/helm_install/)
+
+Pre-requisites: Two Kubernetes clusters named `source` and `destination`.
+ * `StatefulSetStartOrdinal` feature gate is enabled on both clusters
+ * The same default `StorageClass` is installed on both clusters. This
+   `StorageClass` should provision underlying storage that is accessible from
+   both clusters.
+ * A flat network topology that allows for pods to be accessible across both
+   Kubernetes clusters. If creating clusters on a cloud provider, this
+   configuration may be called private cloud or private network.
+
+1. Create a demo namespace on both clusters.
+
+   ```
+   kubectl create ns kep-3335
+   ```
+
+2. Deploy a Redis cluster on `source`.
+
+   ```
+   helm repo add bitnami https://charts.bitnami.com/bitnami
+   helm install redis --namespace kep-3335 \
+     bitnami/redis-cluster \
+     --set persistence.size=1Gi
+   ```
+
+3. On `source`, check the replication status.
+
+   ```
+   kubectl exec -it redis-redis-cluster-0 -- /bin/bash -c \
+     "redis-cli -c -h redis-redis-cluster -a $(kubectl get secret redis-redis-cluster -o jsonpath="{.data.redis-password}" | base64 -d) CLUSTER NODES;"
+   ```
+
+   ```
+   2ce30362c188aabc06f3eee5d92892d95b1da5c3 10.104.0.14:6379@16379 myself,master - 0 1669764411000 3 connected 10923-16383                                                                                                                                              
+   7743661f60b6b17b5c71d083260419588b4f2451 10.104.0.16:6379@16379 slave 2ce30362c188aabc06f3eee5d92892d95b1da5c3 0 1669764410000 3 connected                                                                                             
+   961f35e37c4eea507cfe12f96e3bfd694b9c21d4 10.104.0.18:6379@16379 slave a8765caed08f3e185cef22bd09edf409dc2bcc61 0 1669764411000 1 connected                                                                                                             
+   7136e37d8864db983f334b85d2b094be47c830e5 10.104.0.15:6379@16379 slave 2cff613d763b22c180cd40668da8e452edef3fc8 0 1669764412595 2 connected                                                                                                                    
+   a8765caed08f3e185cef22bd09edf409dc2bcc61 10.104.0.19:6379@16379 master - 0 1669764411592 1 connected 0-5460                                                                                                                                                   
+   2cff613d763b22c180cd40668da8e452edef3fc8 10.104.0.17:6379@16379 master - 0 1669764410000 2 connected 5461-10922
+   ```
+
+4. On `destination`, deploy Redis with zero replicas.
```

Comment:
```suggestion
4. On `destination`, deploy Redis with zero replicas:
```

Reply:
Updated these ordered list bullets to start with action verbs, here and elsewhere